lib/salus/scanners/brakeman.rb

```diff
@@ -6,6 +6,10 @@
 
 module Salus::Scanners
   class Brakeman < Base
+    def self.scanner_type
+      'SAST'
+    end
```

Comment:
If we have a class defining all [constants](https://github.com/coinbase/salus/pull/631#issuecomment-1151534600), then we can write this part as follows:

```ruby
def self.scanner_type
  Salus::ScannerTypes::Types::SAST
end
```